libs/database/src/repositories/inspection-application.repository.ts

```
@@ -356,12 +356,25 @@ export class InspectionApplicationRepository {
   ): Promise<ApplicationInfo[]> {
+    const now = new Date();
+
     return await this.databaseService.inspectionApplication
       .findMany({
         where: {
           inspectionSlot: {
             scheduleUuid,
+            inspectionSlot: {
+              scheduleUuid,
+              ...(includePast
+            ? {}
```

ESLint

[error] Insert `····`

(prettier/prettier)